StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1010CodeFixProvider.cs

```diff
@@ -67,10 +67,15 @@ public override FixAllProvider GetFixAllProvider()
                 if (replacements.Count == 0)
                     continue;
 
-                var transformed = root.ReplaceTokens(replacements.Keys, (original, maybeRewritten) => replacements[original]);
-                Document updatedDocument = context.Document.WithSyntaxRoot(transformed);
-                context.RegisterCodeFix(CodeAction.Create("Fix spacing", t => Task.FromResult(updatedDocument)), diagnostic);
+                context.RegisterCodeFix(CodeAction.Create("Fix spacing", t => GetTransformedDocument(context, root, replacements)), diagnostic);
```

Comment:
:exclamation: Move as much of the logic from this loop to the callback as possible.